kolibri/plugins/facility/assets/src/views/FacilityConfigPage/index.vue

```diff
@@ -218,23 +216,30 @@
           this.toggleSetting('learner_can_edit_password');
         }
       },
-      dismissNotification() {
-        this.$store.commit('facilityConfig/CONFIG_PAGE_NOTIFY', null);
+      updateSettings(action) {
+        this.$store
+          .dispatch(action)
+          .then(() => {
+            // eslint-disable-next-line kolibri/vue-no-undefined-string-uses
+            this.createSnackbar(this.NotificationsStrings.$tr('saveSuccess'));
+            this.copySettings();
+          })
+          .catch(() => {
+            // eslint-disable-next-line kolibri/vue-no-undefined-string-uses
+            this.createSnackbar(this.NotificationsStrings.$tr('saveFailure'));
+            this.$store.commit('CONFIG_PAGE_UNDO_SETTINGS_CHANGE');
```

Comment:
This needs to include the `facilityConfig` namespace, since the action this line was taken from was part of the namespaced `facilityConfig` module.

```suggestion
            this.$store.commit('facilityConfig/CONFIG_PAGE_UNDO_SETTINGS_CHANGE');
```